b2/sync/folder.py

```diff
@@ -193,7 +193,7 @@ def _walk_relative_paths(self, local_dir, b2_dir, reporter, policies_manager):
                 # Check that the file still exists and is accessible, since it can take a long time
                 # to iterate through large folders
                 if is_file_readable(local_path, reporter):
-                    file_mod_time = int(round(os.path.getmtime(local_path) * 1000))
```

Comment:
I would probably have done this as a separate pull request.  It's an independent bug.